content/partials/types/_push_channel.textile

```diff
@@ -41,9 +41,9 @@ h6(#list-subscriptions).
 
 bq(definition).
   java,android: "PaginatedResult":#paginated-result<"PushChannelSubscription":#push-channel-subscription> listSubscriptions(String deviceId, String clientId, String deviceClientId, String channel)
-  objc,swift: listSubscriptions(deviceId: String?, clientId: String?, deviceClientId: String?, channel: String?, callback: ("ARTPaginatedResult":#paginated-result<"PushChannelSubscription":#push-channel-subscription>?, ARTErrorInfo?) -> Void)
+  objc,swift: listSubscriptions(deviceId: String?, clientId: String?, deviceClientId: String?, `channel: String?, callback: ("ARTPaginatedResult":#paginated-result<"PushChannelSubscription":#push-channel-subscription>?, ARTErrorInfo?) -> Void)
 
-"Unsubscribe all devices associated with your device's clientId":/general/push/activate-subscribe#subscribing-client-id from the channel's push notifications.
+Lists push subscriptions on a specified channel name (@channel@) for @clientId@, @deviceId@ or both if @concatFilters@ is set to @true@. This method requires clients to have the "Push Admin capability":/general/push/admin. For more information, take a look at the @GET rest.ably.io/push/channelSubscriptions@ "Rest API":/rest-api.
```

Comment:
Maybe this would be a little clearer:

"Lists push subscriptions on a specified channel (@channel@). You can filter the list of subscriptions by specifying either a @deviceId@ or @clientId@. To include both @deviceId@ and @clientId@ in your filter, set @concatFilters@ to @true@."

Reply:
I took that suggestion and did something similar here: 40063a5d016a91252616fc754d4f1fef997d001c